src/main/java/net/mcreator/ui/dialogs/tools/ToolPackMakerTool.java

```diff
@@ -229,6 +232,7 @@ static void addToolPackToWorkspace(MCreator mcreator, Workspace workspace, Strin
 		shovelTool.repairItems = Collections.singletonList(base);
 		shovelTool.creativeTab = new TabEntry(workspace, "TOOLS");
 		setParametersBasedOnFactorAndAddElement(mcreator, factor, shovelTool, folder);
+		shovelTool.damageVsEntity = (double) Math.round(4.5f * factor);

```

Comment:
Isn't speed missing here?

Reply:
It is set by default to 1 (even if you didn't set it)

Reply:
Those values can change in the UI, so please set this explicitly here.

![image](https://github.com/MCreator/MCreator/assets/16374228/375735ea-f85d-4f5a-887e-c3f051e68bac)

Once again, don't mark resolved by yourself. **Please.**